components/tidb_query_common/src/storage/scanner.rs

```diff
@@ -1,7 +1,8 @@
-// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.key().
```

Comment:
Why change it?

Reply:
It was replaced accidentally, I will revert it.